test/test_autograd.py

```diff
@@ -3132,7 +3132,9 @@ class dont_convert(tuple):
     ('index_fill', (S, S), (0, torch.tensor(0, dtype=torch.int64), 2), 'scalar_index_dim', [0]),
     ('index_fill', (), (0, torch.tensor([0], dtype=torch.int64), 2), 'scalar_input_dim', [0]),
     ('index_fill', (), (0, torch.tensor(0, dtype=torch.int64), 2), 'scalar_both_dim', [0]),
-    ('inverse', (S, S), NO_ARGS, '', NO_ARGS, [skipIfNoLapack]),
+    ('inverse', random_fullrank_matrix_distinct_singular_value(S), NO_ARGS, '', NO_ARGS, [skipIfNoLapack]),
+    ('inverse', random_fullrank_matrix_distinct_singular_value(S, 2, 3),
+     NO_ARGS, 'batched', NO_ARGS, [skipIfNoLapack]),
```

Comment:
Would be nice to add a test case for a tensor of size `(B1, B2, B3, N, N)` (more than one batch dimension)

Reply:
This is already for more than one batch dimensions: the shape tested here is `2 x 3 x S x S`.

Reply:
My bad, I didn't realize